public/js/admin.js

```diff
@@ -911,12 +912,19 @@ var sortAccountsTable = function(type) {
     });
 }
 
-// Toggles the view of the Visibility Checkboxes in the Question-Table View
-var toggleButtonVisibility = function() {
-    if ($('#sw').is(':checked')) {
+
+/**
+ * Toggles the view of the Visibility Checkboxes in the Question-Table View
+ * @param {*} toggle only usesd so the visibily snackbar can turn on display mode
```

Comment:
the {*} should be filled in with the type of the param, like {string} or {object} or whatever. Also on line 1023

Reply:
fixed